### ISSUE TITLE
Make `unistd` as part of if else

### ISSUE THREAD

**Issue by uunnxx** [DinoZ1729/Ray#9]
Hi there!
It's looks so nice <3

Just add `<unistd.h>` to if else statement, and maybe some `readme`'s for linux too ?
```c++
#ifdef _WIN32
#include <windows.h>
#else
#include <unistd.h>
#endif 
```

**Comment by Shroomerian** [comment]
This issue is fixed in my pull at #16. Consider checking it out.

**Comment by hlouzada** [comment]
`unistd` isn't needed with pr #13 see #12
